source/localizable/tutorial/subroutes.md

```diff
@@ -0,0 +1,398 @@
+For Super Rentals, we want to be able to display all rentals and allow the user to view additional information for each.
+To do this, we can generate `rentals` route to display all rentals and a `show` sub-route to display additional information.
```

Comment:
We might want to give a brief description on what a sub-route is, and we we'd want to use it as opposed to just another route.